src/lib/utils/linking.web.js

```diff
@@ -0,0 +1,11 @@
+import { extractQueryParams } from '../share'
+
+class LinkingWeb {
```

Comment:
we also have some utils for Linking  in the master branch (support for open url in the new tab)
please, merge with our changes